### ISSUE TITLE
how to restore training from snapshot model?

### ISSUE THREAD

**Issue by gaopeng-eugene** [DrSleep/tensorflow-deeplab-lfov#1]

**Comment by gaopeng-eugene** [comment]
After using your training script, three files will generate

model.ckpt-16000.data-00000-of-00001  
model.ckpt-16000.meta
model.ckpt-16000.index 

which file should I use to restore training?

**Comment by DrSleep** [comment]
You can just run `--restore_from ./model.ckpt-16000`, it will correctly fetch the right file. 
Note that this is a new writing format of TF since 0.12.

**Comment by gaopeng-eugene** [comment]
there is no restore_from in your training scipt

**Comment by DrSleep** [comment]
oh, I am sorry, I thought this is the TF-resnet repository :)
I have answered you in another topic. Will label it as a bug for now

**Comment by DrSleep** [comment]
I have pushed a commit with the 'restore_from' flag. Please let me know whether you still encounter any problems.
